### PR TITLE
Enhancement/phenotype profile

### PR DESCRIPTION
**Pull request by iimpulse**
AYAYA.

Honestly, theres alot in here. Might be best to merge and test. Unsure.
#161, #181 

Ref for future #254

Any future bugs/enhancements should be a new issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "@vue/cli-plugin-e2e-nightwatch": "^3.9.2",
     "@vue/cli-plugin-eslint": "^3.8.0",
     "@vue/cli-plugin-unit-mocha": "^3.8.0",
-    "@vue/cli-service": "^3.8.0",
+    "@vue/cli-service": "^4.1.2",
```
Comment (kshefchek) [inline package.json]: will bumping a major version affect anything?  wonder if we sync the rest of vue/cli-*

Comment (iimpulse) [reply package.json]: Yeah thought about it.. gonna bump all things major in a new branch. I was scared.

**Comment by kshefchek** [comment]
I think theres a bug in the phenogrid code, for compare it is looking for https://api.monarchinitiative.org/sim/compare, when it should be https://api.monarchinitiative.org/api/sim/compare

**Comment by iimpulse** [comment]
@kshefchek not sure how that happened, but its now fixed if you reinstall the package

**Pull request merged**